generators/pkg/writer/write_test.go

```diff
@@ -0,0 +1,36 @@
+package writer
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestWriteToDisk(t *testing.T) {
+	tempF, err := ioutil.TempFile("", "Temp")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.Remove(tempF.Name())
+
+	err = WriteToDisk("./testdata/spec.yaml", tempF)
+	if err != nil {
+		t.Fatalf("error from 'WriteToDisk': %v", err)
+	}
+
+	got, err := ioutil.ReadFile(tempF.Name())
+	if err != nil {
+		t.Fatal(err)
```

Comment:
Also annotate this and the other testing errors about what failed, so it's easier to see what failed from the go test logs.